plugins/gameobjects/shape/checkbox/Checkbox.d.ts

```diff
@@ -26,11 +26,17 @@ declare class Checkbox extends CheckboxShape {
         config?: Checkbox.IConfig
```

Comment:
If the constructor is described this way, TypeScript won’t understand that the second parameter can be a config.